README.md

```diff
@@ -1,18 +1,29 @@
-=== AMP ===
-Contributors: google, xwp, automattic, westonruter, albertomedina, schlessera, swissspidy, pierlo, johnwatkins0, joshuawold, ryankienstra
-Tags: amp, mobile, optimization, accelerated mobile pages, framework, components, blocks, performance, ux, seo, official
-Requires at least: 4.9
-Tested up to: 5.6
-Stable tag: 2.0.10
-License: GPLv2 or later
-License URI: http://www.gnu.org/licenses/gpl-2.0.html
-Requires PHP: 5.6
+# AMP
 
-The Official AMP Plugin, supported by the AMP team. Formerly Accelerated Mobile Pages, AMP enables great experiences across both mobile and desktop.
+Contributors: google, xwp, automattic, westonruter, albertomedina, schlessera, swissspidy, pierlo,   johnwatkins0, joshuawold, ryankienstra
+Tags: amp, mobile, optimization, accelerated mobile pages, framework, components, blocks, performance,   ux, seo, official  
```

Comment:
Were these blocks of spaces intentional?

![image](https://user-images.githubusercontent.com/16200219/105651575-294afc80-5eaf-11eb-9986-979b8cf0efa8.png)


```suggestion
Contributors: google, xwp, automattic, westonruter, albertomedina, schlessera, swissspidy, pierlo, johnwatkins0, joshuawold, ryankienstra
Tags: amp, mobile, optimization, accelerated mobile pages, framework, components, blocks, performance, ux, seo, official  
```

Reply:
Oops. I saw those extra spaces and in the `Contributors` line and edited your suggestion in https://github.com/ampproject/amp-wp/pull/5807#discussion_r563423666 to remove one of them 😊

Reply:
Fixed other instance in 005b2d3bd